pages/users/add.vue

```diff
@@ -63,13 +63,15 @@
     </section>
 </template>
 <script lang="ts">
-import { Component, mixins } from 'nuxt-property-decorator'
+import { Component, mixins, Watch } from 'nuxt-property-decorator'
 import { email, maxLength, required } from 'vuelidate/lib/validators'
-import BaseMixin from '~/mixins/BaseMixin'
 import { CreateUserRequestModel } from '~/plugins/weavr-multi/api/models/users/requests/CreateUserRequestModel'
 import { UserModel } from '~/plugins/weavr-multi/api/models/users/models/UserModel'
-import ValidationMixin from '~/mixins/ValidationMixin'
 import { Nullable } from '~/global'
+import BaseMixin from '~/mixins/BaseMixin'
+import ValidationMixin from '~/mixins/ValidationMixin'
+
+const Cookie = process.client ? require('js-cookie') : undefined
```

Comment:
Can we maybe globalize this or have it a mixin?

Reply:
> Can we maybe globalize this or have it a mixin?

we have this in other places, maybe could be improvement in another ticket?

Reply:
Sure